### PR TITLE
updating versions in azureastack.psd1 file for the release 1.4.0

### PR DESCRIPTION
**Pull request by bganapa**
## Description

updating versions in azureastack.psd1 file for the release 1.4.0

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@bganapa would you mind resolving the merge conflicts?

**Comment by bganapa** [comment]
rebased, ps-sign here https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/939/

File: src/StackAdmin/AzureStack/AzureStack.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
     RootModule             = "AzureStack.psm1"
 
     # Version number of this module.
-    ModuleVersion          = '1.3.0'
+    ModuleVersion          = '1.4.0'
```
Comment (markcowl) [inline src/StackAdmin/AzureStack/AzureStack.psd1]: We need matching changes to the release notes, correct?  Same for the individual admin module changes.

File: src/StackAdmin/AzureStack/AzureStack.psd1 [hunk 2]
```diff
@@ -121,20 +121,34 @@
...
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Fabric.Admin
+			- Bug fix for returning only a single page in paginated results
+			- Added a cmdlet Add-AzsScaleUnitNode to enable admin to add new scale unit nodes to the azurestack stamp
+			- Added cmdlets New-ScaleOutScaleUnitParametersListObject and New-ScaleOutScaleUnitParametersObject to facilitate the creation scale unit parameter objects
+			* Azs.Gallery.Admin
+			- Bug fix for returning only a single page in paginated results
+			* Azs.InfrastructureInsights.Admin
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Keyvault.Admin
```
Comment (deathly809) [inline src/StackAdmin/AzureStack/AzureStack.psd1]: No bug fixes for KeyVault

File: src/StackAdmin/AzureStack/AzureStack.psd1 [hunk 3]
```diff
@@ -121,20 +121,34 @@
...
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Network.Admin
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Update.Admin
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Subscriptions
+			- Bug fix for returning only a single page in paginated results
+			* Azs.Subscriptions.Admin
```
Comment (deathly809) [inline src/StackAdmin/AzureStack/AzureStack.psd1]: What about Carl's updates?

**Comment by bganapa** [comment]
ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/944

File: src/StackAdmin/AzureStack/AzureStack.psd1 (hunk 1 above)
Comment (bganapa) [reply src/StackAdmin/AzureStack/AzureStack.psd1]: added release notes

**Pull request merged**